Kommunicate-io/Kommunicate-Web-SDK
# Fw-1965-Keep the font-size for all the buttons same in chat widget

### What do you want to achieve?
-Make the font-size for all the buttons same in the chat widget

### PR Checklist


### How was the code tested?

- Add some buttons to your intent using Kompose-Bot Builder.

- Check the font size for all buttons.



![image](https://user-images.githubusercontent.com/22856546/110453573-4993eb00-80ec-11eb-806b-f8707e09618d.png)



![image](https://user-images.githubusercontent.com/22856546/110453754-88c23c00-80ec-11eb-8fbd-e44750d51833.png)



### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
